lib/mindee/http/endpoint.rb

```diff
@@ -104,7 +151,7 @@ def predict_async_req_post(input_source, all_words, close_file, cropper)
 
       # @param job_id [String]
       # @return [Net::HTTPResponse]
-      def document_queue_req_get(job_id)
```

Comment:
revert renaming, the name of these functions should reflect the API route: `/documents/queue`